src/locales/fr.json

```diff
@@ -190,7 +190,8 @@
   },
   "Scan": {
     "takePic": "Prendre en photo",
-    "useExistingPic": "Utiliser une photo existante"
+    "useExistingPic": "Utiliser une photo existante",
+    "selectPic": "Sélectionner une photo existante"
```

Comment:
What's the difference between those to actions? The wording is deferent but the intention seems to be the same?